### DISCUSSION TITLE
How do I increase number of reviewers| remove "up to 1"

### DISCUSSION THREAD

**Discussion by breadbaker** [community/community#23978] [Pull Requests]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/6/260e29836a736b48154269fb82121dce13a92825.png" data-download-href="/uploads/short-url/5qEu8PtGi0T5DJDfkuipcicYyPz.png?dl=1" title="Screen Shot 2021-03-18 at 9.31.05 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/711512/181080484-d71ce71f-5c1f-4a11-b6a2-06c043a01d1e.png" alt="Screen Shot 2021-03-18 at 9.31.05 AM" data-base62-sha1="5qEu8PtGi0T5DJDfkuipcicYyPz" width="690" height="279" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/6/260e29836a736b48154269fb82121dce13a92825_2_690x279.png, https://aws1.discourse-cdn.com/github/original/3X/2/6/260e29836a736b48154269fb82121dce13a92825.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/6/260e29836a736b48154269fb82121dce13a92825.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/6/260e29836a736b48154269fb82121dce13a92825_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-03-18 at 9.31.05 AM</span><span class="informations">710×288 15.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I need more people from my team reviewing prs and I can’t find the setting to increase this limit.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Hi <a class="mention" href="/breadbaker">@breadbaker</a>  - multiple reviewers should automatically be available in a public repository on a free account, or on any repository on a Pro, Team or Enterprise plan.</p>

**Reply by joeflack4** [reply]
@canuckjacq Maybe it's available, but we're having the problem where it is defaulting to 1 reviewer in our organization. I'm looking through organization, team, and repository settings, and I've been googling this as well, and I can't seem to find a way to change it (unless I want to auto-assign reviewers, which I don't).

**Reply by JeremiahTilse** [reply]
Did we ever get a response to this?  I am having the same issue.

**Reply by TugberkS** [reply]
Having the same issue; really frustrating.

**Reply by RicFer01** [reply]
Same issue here, did you get how to do it?

**Reply by TugberkS** [reply]
> Same issue here, did you get how to do it?

If you are using GitHub Teams on a free account, there's no way to increase the number of reviewers. You need to upgrade to a paid plan to use that feature

**Reply by malestroms** [reply]
I have the same issue, please anyone have found a solution for this? I would like to increase the number more than 1 :/ . I have other account with paid plan and it have up to 3, how I can increase it there? @TugberkS

**Reply by tomaskrulis** [reply]
@canuckjacq And where is the setting to increase the maximum number of reviewers?

**Reply by marksark** [reply]
this is stupid

**Reply by phoenixcoded20** [reply]
Maybe they are finding more ways to earn money. This is what happens when an open-source product sits on the lap of a big giant.

**Reply by Jaxolotl** [reply]
that's pretty lame, I moved a repo from a personal private repo to an org private one and now we can only assign 1 reviewer. Cleraly anyone can jump in and review but you cannot assign it. Ridiculous.

**Reply by wg1k** [reply]
My account on a Team plan have this issue with a private repository. According to this answer, it shouldn't. Am I missing something?

**Reply by b-m-tr** [reply]
> Maybe they are finding more ways to earn money. This is what happens when an open-source product sits on the lap of a big giant.

It's not opensource. Git**Lab** is opensource.

**Comment by maryhehehe** [comment]
Hello @breadbaker @joeflack4 @JeremiahTilse @TugberkS , if you're on a public repository on a free account, or on any repository on a Pro, Team, or Enterprise plan, there's repository settings that needs to be turned on so you can choose the number of reviewers it defaults to.

I'd recommend going to the repository settings within branches to set up a branch protection rule. I'd recommend asking an admin in your organization who has access to make changes here. 

I hope this is helpful!

<img width="717" alt="image" src="https://user-images.githubusercontent.com/11730965/208151510-f3125106-cd19-46cf-a80a-32f272a53f00.png">

**Reply by joeflack4** [reply]
Thanks. It's not that I'm looking to require a certain number of approvals, though. I simply want to add multiple reviewers.

**Reply by SaTu07** [reply]
@joeflack4 have you managed to add more than 1 reviewer?

**Reply by joeflack4** [reply]
In some cases, yes, but not all. I am a member of a bunch of different organizations / repos. It's usually the case that I cannot.

**Reply by aerials86** [reply]
I have the same problem. Is it a bug or something else?

**Reply by jss-lpz** [reply]
> I have the same problem. Is it a bug or something else?

Seems like that's by design.

**Comment by edwardcastle** [comment]
Hi, does anyone has increased the number of reviewer of PRs?

**Reply by xia-stan** [reply]
My team had up to 15 reviewers per PR on a free plan up until June 13, 2024. In fact, some of our open PRs that have multiple reviewers still have them listed. No indication about why we can only use one reviewer on new PRs now. Based on @Blobson 's post on March 7th it seems my team was living with a "luxury" that we shouldn't have had.

**Reply by edwardcastle** [reply]
Thanks!! 💫

**Reply by NoobTW** [reply]
Same here, we just noticed that we have not been able to have multiple reviewers/assignees for the last few days.

**Reply by JC3** [reply]
Yeah same situation here, no multiple reviewers for last few days.

**Comment by Blobson** [comment]
It's not allowed on Free plan for private repositories:
![image](https://github.com/community/community/assets/2709384/9b071ca8-231b-44a5-810a-03b4a3ebdcb5)

**Reply by edwardcastle** [reply]
Thanks 🙏

**Reply by JC3** [reply]
Is this a new limitation?

**Reply by xia-stan** [reply]
Based on what I can tell this limitation was rolled out inconsistently to users and organizations. Based on the GitHub docs it's been a limit for at least a year.

**Reply by b-m-tr** [reply]
> Is this a new limitation?

Yes, it definitely wasn't here half a year ago, as we were using multiple reviewers per PR.

**Reply by devmetal** [reply]
The corporate greed is always surprises me
